modules/apps/segments/segments-web/src/main/java/com/liferay/segments/web/internal/configuration/display/SegmentsCompanyConfigurationScreen.java

```diff
@@ -41,7 +41,7 @@
  * @author Cristina González
  * @author Stefan Tanasie
  */
-@Component(immediate = true, service = SegmentsCompanyConfigurationScreen.class)
+@Component(service = ConfigurationScreen.class)
```

Comment:
Hey, you can delete that :(, the FF won't work otherwise

Reply:
Yeahh!!!, the right anotation is @Component(immediate = true, service = SegmentsCompanyConfigurationScreen.class), it will be registered as a ConfigurationScreen in the active method, please, take a look